pyam/utils.py

```diff
@@ -398,7 +398,8 @@ def merge_meta(left, right, ignore_conflict=False):
     if not diff.empty:
         left = left.append(right.loc[diff, :], sort=False)
 
-    return left
+    # remove any columns that are all-nan
+    return left.dropna(axis=1, how="all")
```

Comment:
If there exists tests for this function, would it be possible to add one that would catch this case moving forward (i.e., which would have failed without this new update)? I couldn't find a specific test that would do so that has been added, perhaps missed. Thanks!

Reply:
All of the new ops-tests on the scenario-axis fail if this line is removed, should be enough to ensure that this works going forward...